Finish mobile version

pulgosodog opened the pull request:
Implementations on this pull request following the **Portfolio: finish mobile version** activity:

1. Add works sections to index.html
2. Add About myself section to index.html
3. Update CSS styles in style.css file.
4. Add PNG files to repository

pulgosodog committed:
Add works section

Added works sections cards

pulgosodog committed:
Fix spacing on the card button

pulgosodog committed:
Add about myself section, update style.css

pulgosodog committed:
Finish the about myself section

Added the abilities and added a ul under language ability

pulgosodog committed:
Update style.css

Fix mark issues by the linters

AdnanOlarmmi commented on index.html:
```diff
@@ -46,6 +47,191 @@ <h1 class="margin">I'm Gabriel. <br> Glad to see <br> you!</h1>
+                    <p>javascript</p>
+                </li>
+            </ul>
+            <div class="div-button"><button class="card-item">See project</button></div>
```
- [ ] Kindly ensure that your buttons also follow the design guidelines, such as `enabled`, `hover`, `pressed`, and `disabled`, so as to have a pixel-perfect design like you'll soon be doing in your job 🙂.

- Your design
![mobile](https://user-images.githubusercontent.com/41163571/235327622-029224c0-6b91-402e-8a24-3be1fb2d50cf.gif)

- Figma design
<img width="446" alt="right" src="https://user-images.githubusercontent.com/41163571/227707940-ae12a5de-15d8-4b79-b06b-a16bfdafd017.png">

- For all the buttons.

pulgosodog replied:
Got it! I'm going to add the interactions to the buttons.

pulgosodog replied:
Done! ✔
I implemented button interactions acroding to the figma desing.

AdnanOlarmmi commented on style.css:
```diff
@@ -1,37 +1,61 @@
   background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
   display: flex;
```
- [ ] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` and assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.

pulgosodog replied:
I'm going to do my best into incorporating this to my code. 🙌💯

pulgosodog replied:
Issue addressed, I refactored the code! 👨‍💻

AdnanOlarmmi commented on style.css:
```diff
@@ -68,8 +96,164 @@ ul {
+}
+
+li.languages {
+  display: flex;
```
- [ ] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` and assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.

pulgosodog replied:
Noted! I'll be very carefull not to break my code while doing so, thanks you for pointing this out!

pulgosodog replied:
Issue addressed, I refactored the code! 👨‍💻

AdnanOlarmmi reviewed (changes requested):
Hi @pulgosodog, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

![You're Almost there, funny one eyed alien](https://user-images.githubusercontent.com/41163571/217957948-6d8d9f33-125c-4724-954f-330eed60533c.gif)

## Highlights
- Nice overall design.
- Excellent usage of grid and flex box properties respectively.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

pulgosodog committed:
Refactor style.css and update index.html

Add :
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}

classes to the CSS document and implement them on html following what the code reviewer pointed out to refactor the code.

Efiamotu-1 commented on style.css:
- [ ] the previous reviewer requested changes for the button interactions which have still not been implemented and as such i will be requiring you to do so and giving you a practical example of how to go about it. 
- first off you need to give every button a unique class name eg `btn` and then style this btn class using pseudo selctors such as `:hover. :active` and apply the various background color and color from the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=bLjL2tJJSSkEfkRj-0) to these styling.
- so this is what your css code would look like
```
.btn:hover {
  background-color: red;
  color: white
}
```
- [ ] Note the example below is only for hover and i have only used random color do  well to check the figma for the correct color as shown below
![req-t-1-btn-int](https://user-images.githubusercontent.com/89055502/235329283-1fda78dd-7440-4f5a-a1e1-06744acf96d4.png)

pulgosodog replied:
Oh! My bad, I'm sorry, I haven't push, I have the changes locally. Let me push them to the github repository.

pulgosodog replied:
Local changes pushed!

Efiamotu-1 commented on style.css:
- [ ] you are not using the correct font size for as shown from the figma design please correct this.
- figma design
![cp-f](https://user-images.githubusercontent.com/89055502/235329342-a49fe236-00c0-43b2-b17d-8774a4e1a4f5.png)
- your design
![canopy-s](https://user-images.githubusercontent.com/89055502/235329373-d93f6792-e954-423e-a5e2-ad485077643c.png)

pulgosodog replied:
Hello! I'm using its equivalent on **vw** with two decimals as pixels would not be reactive.
Using the vw equivalent: 
![image](https://user-images.githubusercontent.com/115130832/235330273-636da999-ca43-440a-8fb5-2265caa03ef3.png)
![image](https://user-images.githubusercontent.com/115130832/235330367-841abbd2-caeb-4334-8950-b61479e1e4c6.png)

![123](https://user-images.githubusercontent.com/115130832/235330378-f4e69b08-031c-46c5-9341-52bbd4a0b947.JPG)

Efiamotu-1 replied:
@pulgosodog Okay good, your next reviewer will take note of this, just go ahead and implement the other changes so you can be approved

pulgosodog replied:
Using the describe Figma font size in pixels:
![image](https://user-images.githubusercontent.com/115130832/235330426-7b33a7cc-ed33-405b-81e2-5fc7dd443bb3.png)
![image](https://user-images.githubusercontent.com/115130832/235330443-6faf562c-83c1-4f0d-9068-15762cc3e84f.png)
![1234](https://user-images.githubusercontent.com/115130832/235330450-2d1d4889-9d62-4557-ad54-d6ca03ade902.JPG)
![image](https://user-images.githubusercontent.com/115130832/235330481-03f29971-315c-4029-ac22-82418feb5323.png)

pulgosodog replied:
Addressed! 🥳🙌
I wanted to leave it just as **3.46vw** as I was afraid the Linters will marked it bad because of having more than 2 decimals.  Changing the font-size to **3.467vw** for it to be more consistent acording to the figma project.
![image](https://user-images.githubusercontent.com/115130832/235330643-c38c5359-c0df-4aed-a2cc-ee2b72c70797.png)
![image](https://user-images.githubusercontent.com/115130832/235330663-2d082e66-9765-4641-a015-63320a3b10ba.png)
![image](https://user-images.githubusercontent.com/115130832/235330651-bd27f90f-f433-4db5-9d7f-ac8c5ba63c62.png)

Efiamotu-1 commented on style.css:
- [ ] please work on the alignment of the paragraph and the technologies as you can see from the figma design they all start on the same line but for your design ther is a bit of margin from the html please kindly correct this before your ext review request :+1: 
- figma design
![f-ct](https://user-images.githubusercontent.com/89055502/235329460-e078b8a1-d214-4a03-83fb-bf0b31409422.png)
- your design
![s-ct](https://user-images.githubusercontent.com/89055502/235329469-62243108-2380-41c0-b9d1-61a3a149677d.png)

pulgosodog replied:
Addressed! 👨‍💻🙌I also added some padding to the top and bottom, got some letters of the **p** to upper case, and fixed the font color of the **UL** "categories" **LI**s.

Efiamotu-1 reviewed (changes requested):
Hi @pulgosodog , 

![](https://camo.githubusercontent.com/a06454929d29b512682cd8b375641b59076b9e5fe1a367808e685e5bcb818de3/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f76312e59326c6b505463354d4749334e6a45784e32466d5957466c5a6a45794d7a49314f4455784d5455775a44686b595459355932526c4e6a4d334e7a4e6c4f574530596a4a684d695a6c634431324d563970626e526c636d35686246396e61575a7a583264705a6b6c6b4a6d4e305057632f7a3370696f6b77663057504838314d4f68752f67697068792e676966)

Great work on making the changes requested by a previous reviewer 👏 Your Code is Awesome! 💪
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Especially if that's is your first project 1️⃣. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better job** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [![@Efiamotu-1](https://img.shields.io/github/followers/Efiamotu-1?label=Habeeb-Efiamotu-Musa&style=social)](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[![Habeeb Efiamotu Musa](https://img.shields.io/badge/linkedin-@Habeeb-crimson)](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [![@EFYAMOTU](https://img.shields.io/twitter/follow/omarramoun?label=Habeeb-Efiamotu-Musa&style=social)](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [![Habeeb Efiamotu Musa ](https://img.shields.io/badge/slack-@Habeeb-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

pulgosodog committed:
Add button interactions

Add button:active, button:hover, button:disabled interactions.

pulgosodog committed:
Update font size of h4

Added a decimal number to be consistent with the figma project

pulgosodog committed:
Add Margin-left:0 to the first UL "categories" LI

Also add some padding to the top and bottom, got some letters of the p to upper case, and fixed the font color of the UL "categories" LIs.
Change the button color to transparent.

pulgosodog committed:
Fix linter mark errors

didierganthier reviewed (approved):
Hi @pulgosodog ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

<img src="https://media.giphy.com/media/l9CSJHK88kMTe/giphy.gif" />

## Highlights
- Followed design guidelines ✔️ 
- Used semantic HTML tags ✔️ 
- Followed orrect GitHub flow ✔️ 

## Optional Suggestions 💡
- N/A

Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉
